---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Global Value Chain dalam bumbu gado-gado"
subtitle: ""
summary: ""
authors: [admin]
tags: [perdagangan, Global Value Chain, Python, UNCOMTRADE]
categories: [perdagangan, Global Value Chain, Python, UNCOMTRADE]
date: 2021-08-31T23:27:21+10:00
lastmod: 2021-08-31T23:27:21+10:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: "Produk dengan kode HS 21039019"
  focal_point: ""
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
---

Hari ini saya sarapan gado-gado. Menurut saya, gado-gado (dan bumbu kacang secara umum) adalah sebuah fenomena yang luar biasa. Meskipun kacang pertama kali [diperkenalkan oleh Portugis yang dibawa dari Mexico](https://en.wikipedia.org/wiki/Peanut_sauce), Masyarakat Indonesialah yang pertama kali membuat bumbu kacang. Bumbu kacang ini sungguh _versatile_ karena bisa dicampur apa aja. Dikasih sayur, bisa. Kasih gorengan, enak. Nasi uduk, hayu. Buat bumbu ayam bakar, ya bisa (alias jadi sate).

Orang bule juga ternyata sangat menyukai bumbu kacang. Seperti kita ketahui, orang bule kalo sarapan suka pake roti yang diolesin selai kacang. Tentunya sungguh wajar jika mereka juga suka bumbu kacang. Daripada salad dressing a la eropa, jelas lah enakan gado-gado. wkwkwk. Kesukaan rang bule akan bumbu kacang (ditambah dengan banyaknya orang Asia dan orang Indonesia di Australia) memberikan skala ekonomi yang cukup untuk importir bumbu kacang.

Tentunya sebagai insan yang ga bisa masak, saya lebih suka pake bumbu kacang yang udah jadi. Praktis. ga perlu ngulek-ngulek. Ntar kalo bikin sendiri mah udahlah nguleknya capek, makan waktu, kagak enak pula! Udahla mending kita beli bumbu jadi aja. Praktis dan udah pasti rasanya oke.

Saya suka beli bumbu ini, yang namanya "Enak Eco", beli di toko Vietnam deket rumah. Harganya sekitar A$4. Mantap ya. Export quality wkwkw. Diekspor dari mana sih benda ini?

![gado2](gado2.jpg)


Ternyata bumbu ini adalah karya anak bangsa. Made in Malang, tepatnya diproduksi oleh PT Enak Jaya Makmur. Diimpor oleh Sony Trading di New South Wales.

## Cabe lalu gado-gado: peran Global Value Chain

Beberapa waktu lalu, saya sempet bikin postingan tentang [impor cabe](https://krisna.or.id/post/cabe) yang ternyata lumayan banyak peminatnya. Memang ketika itu situasinya adalah harga cabe sedang rendah, sehingga petani merugi. Di postingan tersebut juga saya memperlihatkan bahwa cabe mentah sudah dibatasi impornya, sementara cabe kering impornya bebas.

Katanya sih cabe mentah dikonsumsi oleh masyarakat, sementara cabe kering dikonsumsi oleh industri. Meskipun ternyata ga juga, akan tetapi tidak dapat kita pungkiri bahwa industri berperan besar dalam mengimpor cabe, terutama melihat betapa [besarnya pertumbuhan industri makanan dan minuman](https://www.wartaekonomi.co.id/read323825/industri-makanan-minuman-masih-jadi-andalan-di-2021) di Indonesia.

Menggunakan produk impor untuk diolah di dalam negeri menjadi produk ekspor bukanlah hal baru. Sejak zaman di mana banyak negara melakukan perjanjian dagang, adalah hal lumrah bagi perusahaan di suatu negara untuk menggunakan bahan baku dari negara lain. Seperti halnya coklat belgia yang cokelatnya datang dari pantai gading, gado-gado Indonesia pun cabenya datang dari negara lain. Vietnam, negara yang sejatinya berteknologi rendah, dapat menjadi eksportir ponsel pintar besar dunia karena mengimpor komponen. Ekonom memakai istilah "Global Value Chain" (GVC) untuk mendeskripsikan fenomena ini.

Di postingan cabe itu, kita juga jadi tahu bahwa ada negara lain yang sanggup bikin cabe lebih murah daripada Indonesia. Akan tetapi, mereka ga bisa bikin gado-gado seenak kita lho (mungkin). Hahaha. Dengan cabe yang murah, maka bumbu gado-gado juga bisa jadi murah. Akibatnya, produk ini jadi laris tidak hanya di pasar dalam negeri, tetapi juga di pasar luar negeri.

Tanpa berpanjang lebar lagi, mari kita coba cek 'kandungan internasional' di dalam bumbu gado-gado impor ini. haha.

## Ingredients dengan bantuan UN COMTRADE

Tentunya saya ga begitu paham struktur industri bumbu gado-gado. Karena itu, kita akan pake kira-kira aja. Dari bungkus belakang bumbu ini, dapat dilihat bahwa bumbunya antara lain adalah kacang tanah, cabe (capsicum), coconut sugar, bawang putih, bawang merah, garem, gula, dan pengawet (hiii baru ngeh ada pengawetnya tidaaaak).

Dari list tersebut, saya tinggal googling HS Code, untuk mendapatkan kode HS buat kita tarik datanya dari UN COMTRADE. Di sini kita sekali lagi harus bikin _caveat_ wkwkw. Pertama, Ga ada informasi bahan-bahan ini dibeli oleh PT Enak Jaya Makmur dalam bentuk apa. Misalnya, apa kacang tanahnya murni yg masih ada kulitnya? Atau udah dikupas dan disangrai? Cabe pun demikian. Basah? Kering? Bubuk? Kode HS nya beda-beda. haha! Karena tidak ada informasi ini, maka saya melakukan sedikit spekulasi. Jadi jangan terlalu percaya sama kode HS yang akhirnya saya pake wkwkwkw. Jika anda lebih tahu, kode yg saya pake di postingan ini bisa direplikasi dengan mudah, jadi bisa coba versi anda sendiri. Atau lebih baik lagi, kasih tau saya resep yg benernya melalui [twitter](https://twitter.com/iMedKrisna) wkwkwk.

Seperti halnya postingan cabe, kita tarik data-data yang diperlukan pake API dari [UN COMTRADE](https://comtrade.un.org/Data/).

In [73]:
# cari api key-nya di https://comtrade.un.org/Data/
api_key='/api/get?max=502&type=C&freq=A&px=HS&ps=2019&r=360&p=0&rg=2%2C1&cc=200811%2C170290%2C070320%2C070310%2C071220%2C250100%2C291619%2C170111%2C090421%2C170114'
url='https://comtrade.un.org/'+api_key+'&fmt=csv'
gado2=pd.read_csv(url)
gado2=gado2[['Year','Trade Flow','Commodity Code', 'Commodity',
           'Netweight (kg)', 'Trade Value (US$)']]

Setelah kita download datanya, lalu saya coba perlihatkan deskripsi dari komoditas yang saya pake. Itu 4 komoditas pertama harusnya ada 0 di depannya (jadi 070310, 6 digit) tapi saya terlalu malas untuk konversi jadi biar aja jadi 5 digit wkwkw.

In [75]:
pd.set_option('display.max_colwidth', None) # show full content of a column
desc=gado2[['Commodity Code','Commodity']] # ambil cuma kode dan nama barang
desc.drop_duplicates() # print

Commodity Code  \
0            70310   
2            70320   
4            71220   
6            90421   
8           170114   
10          170290   
12          200811   
14          250100   
16          291619   

                                                                                                                                                                                                                Commodity  
0                                                                                                                                                           Vegetables, alliaceous; onions and shallots, fresh or chilled  
2                                                                                                                                                                        Vegetables, alliaceous; garlic, fresh or chilled  
4                                                                                                                             Vegetables; onions, whole, cut, sliced, broken or in powder but not further prepared, dried  
6                                                                                                                                      Spices; fruits of the genus Capsicum or Pimenta, dried, neither crushed nor ground  
8                                                               Sugars; cane sugar, raw, in solid form, other than as specified in Subheading Note 2 to this chapter, not containing added flavouring or colouring matter  
10                                                              Sugars; n.e.c. in heading no. 1702, including invert sugar and other sugar and sugar syrup blends containing, in the dry state, 50% by weight of fructose  
12                                                                                                                            Nuts; ground-nuts, whether or not containing added sugar, other sweetening matter or spirit  
14                                                                                                     Salt (including table salt and denatured salt); pure sodium chloride whether or not in aqueous solution; sea water  
16  Acids; unsaturated acyclic monocarboxylic, cyclic monocarboxylic, their anhydrides, halides, peroxides and peroxyacids; their halogenated, sulphonated, nitrated or nitrosated derivatives, n.e.c. in item no. 2916.1

Seperti dapat dilihat, deskripsinya agak menyebalkan wkwk. Saya sertakan juga bubuk bawang di situ. Untuk cabe, saya pake cabe kering. Saya ga gitu tau Coconut sugar pake HS yg mana, tapi jadinya di sini saya pake 170290. Feel free to disagree with my choices hahaha.

Oh iya satu lagi, saya pake data tahun 2019. 2020 ada tapi takut rusak gara-gara COVID. 2019 harusnya ga jauh beda dengan business as usual.

Okeh, setelah dapat barang-barang yang mau dilirik, mari kita tampilkan ekspor dan impornya bahan-bahan ini.

In [89]:
## Tarik ekspor doang
x1=gado2.query('`Trade Flow`=="Export"').reset_index()
x1=x1[['Commodity Code','Netweight (kg)','Trade Value (US$)']]
x1=x1.rename(columns={"Netweight (kg)": "Ekspor (kg)",
                                          "Trade Value (US$)": "Ekspor (US$)"})
## Tarik impor doang
m1=gado2.query('`Trade Flow`=="Import"').reset_index()
m1=m1[['Commodity Code','Netweight (kg)','Trade Value (US$)']]
m1=m1.rename(columns={"Netweight (kg)": "Impor (kg)",
                                          "Trade Value (US$)": "Impor (US$)"})
## Gabung dan bikin tabelnya enak dilihat
gabung=pd.merge(x1,m1,on='Commodity Code',how='outer')
gabung['Product description']=['Onion/Shallot','Garlic','Onion Powder','Dried Chili','Cane Sugar','Coconut Sugar','Ground-nuts','salt',
                 'Perservatives']
gabung=gabung[['Commodity Code','Product description','Ekspor (kg)','Impor (kg)','Ekspor (US$)','Impor (US$)']]
## Print dengan format thousand separated by comma
gabung.style.format({"Ekspor (kg)": "{:,.0f}", "Impor (kg)": "{:,.0f}"
                    , "Ekspor (US$)": "{:,.0f}", "Impor (US$)": "{:,.0f}"})

Seperti dapat dilihat pada tabel di atas, hampir semua bahan-bahan yang saya gunakan di sini memiliki impor yang jauh lebih tinggi daripada ekspornya. Dengan kata lain, kemungkinan kandungan internasionalnya (setidaknya dari sisi bahan baku) cukup tinggi. Bumbu gado-gado ternyata adalah produk GVC!! Kita beli bahan baku dari negara lain, kita olah, lalu kita jual lagi ke berbagai pasar di belahan dunia. Di konsep GVC, kita bisa bilang PT Eka melakukan "Backward participation". Ga salah emang kalo Indonesia dibilang surganya makanan!



But at what cost?



## Kebijakan pembatasan impor



Diantara bahan-bahan yang saya pake di atas, sebagian mengalami pembatasan impor, dan sebagian lain dapat diimpor secara bebas. Bawang merah, bawang putih, gula, kacang dan garam adalah barang-barang yang impornya diregulasi. Dengan kata lain, pemerintah menerapkan kuota impor, di mana hanya perusahaan tertentu yang ditunjuk oleh negara (biasanya BUMN) yang boleh impor.



Pembatasan ini dirasa perlu karena Pemerintah berupaya melindungi petani, peternak dan nelayan di Indonesia dari kompetisi internasional. Indonesia memang memiliki beragam masalah dalam memajukan sektor pertanian. Solusi paling mudah dan populer adalah dengan melakukan pembatasan impor. Kenapa bisa populer ya? Ntahlah.



Sayangnya, meski memiliki tujuan mulia, tidak semua hal yang diserahkan negara memiliki pengelolaan yang baik. Diantara kasus korupsi kuota impor yang paling terkenal mungkin adalah [kasus suap kuota daging sapi](https://nasional.tempo.co/read/458101/suap-daging-pks-begini-awal-mulanya) yang melibatkan petinggi penting sebuah parpol. Namun kasus korupsi juga tidak jauh dari bumbu gado-gado yang saya beli. Diantaranya kasus suap [kuota bawang putih](https://nasional.tempo.co/read/1290541/bin-dituduh-punya-jatah-dalam-kasus-suap-impor-bawang-putih/full&view=ok), yang akhirnya pelakunya dipenjara meskipun di bawah tuntutan. Di samping itu, ada juga kasus [korupsi gula](https://nasional.tempo.co/read/1341932/suap-impor-gula-kpk-tuntut-eks-bos-ptpn-iii-6-tahun-penjara) yang melibatkan eks bos BUMN gula, dan ada juga tersangka [pengadaan garam impor](https://tirto.id/korupsi-dan-sengkarut-pengelolaan-garam-cqvi). Suap-menyuap ini, [menurut KPPU](https://bisnis.tempo.co/read/1454576/kppu-ingatkan-pemerintah-soal-potensi-rente-impor-garam/full&view=ok), diakibatkan oleh buruknya transparansi penunjukkan perusahaan yang berhak mendapatkan kuota.



Tidak semuanya kasus korupsi, kadang pengendalian perdagangan juga tidak sempurna dan mengakibatkan adanya barang ilegal atau palsu, seperti di kasus [bawang merah](https://www.merdeka.com/uang/pemerintah-dituding-berperan-ciptakan-kasus-bawang-merah-palsu.html).



Salah satu penelitian yang mencoba mendokumentasikan hal-hal ini ditulis oleh kolega saya Felippa Amanta yang bisa dibaca di [sini](https://repository.cips-indonesia.org/media/341330-dampak-negatif-kebijakan-perdagangan-non-d6b37e99.pdf).



Namun pada prinsipnya, kebijakan pembatasan impor sejatinya adalah sebuah keberpihakan. Produsen bahan baku (petani, nelayan dan peternak) adalah pihak yang diuntungkan dari dilakukannya kebijakan seperti ini. Koruptor juga untung sih hahaha. Yang rugi siapa? Tentunya industri manufaktur dan konsumen secara umum. Dengan akses yang sulit terhadap bahan baku, maka akan sulit manufaktur kita berkembang. 



Kalau ditanya ke orang ekonomi, maka pastinya jawabannya adalah soal efisiensi. Mana yang lebih memberikan nilai tambah tinggi? proteksi petani untuk konsumsi domestik, atau mengimpor bahan baku dan menghasilkan produk  yang bisa diekspor? Biasanya sih keterbukaan perdagangan memberikan hasil yang lebih efisien. Bahkan, kalo kata dosen saya di sini, akan lebih baik jika "loser" dari perdagangan kita kompensasikan dari "gain" yang didapatkan dari perdagangan bebas. Dengan kata lain, kita pajakin eksportir (looking at you CPO dan batubara), lalu uangnya kita gunakan untuk membiayai program-program pemerintah yang beriorientasi pada pelayanan publik, seperti kesehatan, pendidikan dan infrastruktur. Di samping itu, fasilitasi untuk mempermudah perpindahan tenaga kerja dari yang kerja di sektor "loser" untuk pindah ke sektor "gainer" bisa juga jadi opsi. Sebenarnya anak muda yang memilih jadi produsen manufaktur daripada petani adalah sebuah awal menuju perubahan struktur ekonomi yang tinggi agrikultur menuju tinggi manufaktur.

Bisa jadi juga program-program seperti itu lumayan kompleks dan merepotkan. Program-program tersebut juga bisa aja dikorup juga. Di samping itu, pembatasan impor lebih mudah, dan disenangi tidak hanya masyarakat tetapi juga elit politik dan birokrat.

Mungkin juga ada alasan yang lebih fundamental. Lebih ideologis. Apa artinya efisien jika identitas kita sebagai negara agraris terkikis? Jika identitas sebuah bangsa adalah berdasarkan produk yang dia ekspor, apa mau kita berubah dari negara agraris jadi negara CPO? Atau negara pembuat gado-gado? Semua demi efisiensi?



Yha pada akhirnya ini adalah pertanyaan politik, bukan ekonomi. Ekonomi mah cuma sebuah alat aja, untuk membantu para pembuat visi untuk mencapai visi tersebut. Jika masyarakat memilih untuk menjaga identitas sebagai negara agraris dengan mengabaikan efisiensi, maka tentu itu wajar-wajar saja.



Sekian postingan ini! Mari kita selalu optimis menjadi bangsa yang besar, maju, dan berdaya saing! Dan tetap makan gado-gado!
